.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Problems
+<!--- What problems do you want to solve -->
+<!--- Ex: API get product list takes long time -->
+- ...
+
+# Purposes
+<!--- Solutions to be used to solve the problems -->
+<!--- Ex: Add mechanism using Redis -->
+- ...
+
+# Changes
+<!--- Please describe the changes or the solution here -->
+<!--- Ex: Call redis inside get product handler function, before querying to database -->
+- ...
+- ...
+
+# Checklist
+These are the checklists to make sure your PR is ready to be reviewed.
+- [ ] Unit testing
+- [ ] DB changes is migrated on staging
```

Comment:
I think the word "on staging" is a bit confusing to me Mas, since we are using auto migrate, so whenever it is deployed to staging, it will automatically be migrated there. How about changing the sentence to "DB changes is successfully migrated" or simply just removing the "on staging" part?

Reply:
changed. thanks